runtime/stdlib/crypto.go

```diff
@@ -153,7 +156,11 @@ func newCryptoContractHashFunction(hasher CryptoHasher) interpreter.FunctionValu
 			}
 			hashAlgorithm := hashAlgorithmStringValue.Str
 
-			digest := hasher.Hash(data, hashAlgorithm)
+			digest, err := hasher.Hash(data, hashAlgorithm)
+			if err != nil {
+				panic(fmt.Errorf("hash: hashing failed: %w", err))
```

Comment:
Just panic with the error

```suggestion
				panic(err)
```